strawberryfields/gbs/plot.py

```diff
@@ -343,3 +344,76 @@ def points(
         f = go.Figure(data=[p], layout=layout)
 
     return f
+
+
+def spectrum(
+    energies: list, gamma: float = 100.0, xmin: float = None, xmax: float = None
+):  # pragma: no cover
+    """Plots a spectrum based on input sampled energies.
+
+    Args:
+        energies (list[float]): a list of sampled energies
+        gamma (float): parameter specifying the width of the Lorentzian function
+        xmin (float): minimum limit of the x axis
+        xmax (float): maximum limit of the x axis
+
+    Returns:
+         Figure: spectrum in the form of a histogram of energies with a Lorentzian-function curve
+    """
+    try:
+        import plotly.graph_objects as go
+    except ImportError:
+        raise ImportError(plotly_error)
+
+    if xmin is None:
+        xmin = min(energies) - 1000.0
+    if xmax is None:
+        xmax = max(energies) + 1000.0
+
+    bins = int(max(energies) - min(energies)) // 5
+    bar_width = (max(energies) - min(energies)) * 0.005
+    line_width = 3.0
+
+    h = np.histogram(energies, bins)
+
+    X = np.linspace(xmin, xmax, int(xmax-xmin))
+
+    L = 0
+    for e in energies:
+        L += (gamma / 2)**2 / ((X - e)**2 + (gamma / 2)**2)
+
+    text_font = dict(color="black", family="Times New Roman")
```

Comment:
What other fonts are available? It would be great if we could use Computer Modern, the latex font. If not, we should probably pick something that resembles it

Reply:
It seems that we do not have many choices.